tasks/main.yml

```diff
@@ -1,34 +1,5 @@
 ---
-- name: Ensure we have pantheon_version
-  fail:
-    msg: You must set "pantheon_version" for this role to run
-  when: pantheon_version is not defined
+- name: Pantheon has moved to Hyperledger Besu https://galaxy.ansible.com/pegasyseng/hyperledger-besu
+  debug:
```

Comment:
this should be `fail` not `debug`.  Doing the update must _fail_ loudly so that it will be seen

Reply:
Good point, updates